backend/src/test/java/com/woowacourse/momo/fixture/DateFixture.java

```diff
@@ -0,0 +1,28 @@
+package com.woowacourse.momo.fixture;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+import lombok.RequiredArgsConstructor;
+
+@SuppressWarnings("NonAsciiCharacters")
+@RequiredArgsConstructor
+public enum DateFixture {
+
+    _6월_30일(2022, 6, 30),
+    _7월_1일(2022, 7, 1),
+    _7월_2일(2022, 7, 2);
+
+    private final int year;
+    private final int month;
+    private final int day;
+
+    private LocalDate instance;
+
+    public LocalDate getInstance() {
+        if (Objects.isNull(instance)) {
```

Comment:
Objects.isNull() 의 주석을 보면 아래와 같이 적혀있어요.

> API Note:
This method exists to be used as a java.util.function.Predicate, filter(Objects::isNull)

내부 코드를 보면 단순하게 null 체크를 하지만 용도에 맞춰서 사용하는건 어떨까요?

```suggestion
        if (instance == null) {
```

Reply:
좋습니다~~!!😆